ReadME.md

```diff
@@ -31,10 +49,34 @@ A service that monitors ZKSync and Ethereum networks for governance events and g
 - `npm run build` - Build both frontend and backend
 - `npm run clean` - Clean all dependencies and build artifacts
 
+## Docker Commands
+
+- `docker build -t zksync-rss-backend-init -f Dockerfile.init .` - Build initial setup image
+- `docker run zksync-rss-backend-init` - Run initial setup
+- `docker build -t zksync-rss-backend .` - Build main service image
+- `docker run zksync-rss-backend` - Run the block processing service
+
 ## Project Structure
 
+```
 ├── packages/
-│ ├── backend/ # Backend service
-│ └── frontend/ # Frontend application
+│   ├── backend/    # Backend service
+│   └── frontend/   # Frontend application
+├── Dockerfile      # Docker configuration for recurring process
+├── Dockerfile.init # Docker configuration for initial setup
 ├── package.json
-└── README.md
\ No newline at end of file
+└── README.md
+```
+
+## Environment Variables
+
+- **./packages/frontend/.env** Place your gcp rss file link here ex: https://storage.googleapis.com/zksync-rss/rss/feed.xml
+- **./packages/backend/.env** refer to the sample
+- 
+
+## Notes
+- The initial setup (`process-historic-blocks`) should be run only once when setting up the service
```

Comment:
Does historic blocks have the old event blocks hardcoded? Asking because there has been some new events and there should probably be some explanation on how to find and add them.

Reply:
yes, it's hardcoded, i have all events covered up to a week ago, the process would be finding events, getting their blocks, then adding it into the entry file